package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-override-color-scheme",
-  "version": "1.0.3",
```

Comment:
@shikhin21 I think it should be 2.0 because raising `minSdkVersion` is a breaking change?